src/Anonymization/Anonymizer/Core/PasswordAnonymizer.php

```diff
@@ -22,6 +23,8 @@ class PasswordAnonymizer extends AbstractAnonymizer
     #[\Override]
     protected function validateOptions(): void
     {
+        MissingDependencyException::check('symfony/password-hasher', PasswordHasherFactory::class);
```

Comment:
Doesn't PHP grumble right from `PasswordHasherFactory::class` if the component is missing ?

Reply:
Nope, class names are handled OK, it will crash only if you try to use it (new instance, static method, constant...), but the `Foo::class` expression is just a string, containing the whole class name as found in the `use` statement.